script/bootstrap

```diff
@@ -4,6 +4,18 @@
 set -e
 set -x
 
+#!/bin/bash
```

Comment:
Is this shebang needed? It seems to be the same as the one on the top level of the file.